core-bundle/src/EventListener/StoreRefererListener.php

```diff
@@ -66,11 +66,7 @@ public function __invoke(ResponseEvent $event): void
             return;
         }
 
-        if ($this->scopeMatcher->isBackendRequest($request)) {
-            $this->storeBackendReferer($request);
-        } else {
-            $this->storeFrontendReferer($request);
-        }
+        $this->storeBackendReferer($request);
```

Comment:
I don't think you want to store the back end referer in the front end …

Reply:
True. 😊 Fixed in 0e280359c2f2e3c267e6409507e999c50c9e4732.